Feature/004 display offer

paulinaurbas opened the pull request.

paulinaurbas committed:
001-project-structure
added base project

paulinaurbas committed:
001-project-structure

change languages, bloc init

paulinaurbas committed:
001-project-structure

added get it

paulinaurbas committed:
002-login-register

added bloc, logic, redirections, repositories

paulinaurbas committed:
Merge branch 'main' into feature/002-login-register

paulinaurbas committed:
002-login-registration

Updated readme

paulinaurbas committed:
002-login-register

added login screen

paulinaurbas committed:
002-login-register

added login page with registration button

paulinaurbas committed:
002-login-registartion

added registration

paulinaurbas committed:
002-user-see-offer

added login local save

paulinaurbas committed:
002-login-regitser

added calls to backend + logout

paulinaurbas committed:
002-login-register

added translation

paulinaurbas committed:
002-login-registration

added files

paulinaurbas committed:
004-display-offer

corrected background, added filthering offer view, single card, settings page

paulinaurbas committed:
004-display-offer

added logic to screens, prepared for api calls

paulinaurbas committed:
004-display-offer

corrected colors

paulinaurbas committed:
004-display-offer

details screen + bloc

paulinaurbas committed:
004-display

added galery

paulinaurbas committed:
004-display-offer

added details screen

paulinaurbas committed:
Merge branch 'main' into feature/004-display-offer

franpog859 reviewed (commented):
1. ![Screenshot 2021-12-18 at 13 39 10](https://user-images.githubusercontent.com/32485355/146641431-aec813d1-2991-4e65-b09a-bead5d0dedd4.png)
I got this error. I think it's something wrong with styles

1. ![Screenshot 2021-12-18 at 13 40 22](https://user-images.githubusercontent.com/32485355/146641447-51508763-1320-4f8e-bf45-4f0527fcdead.png)
I'm not sure if it's a bug or not, but while clicking on the photo in `More photos` section, nothing happens in the app but some diagnostic error shows up in the IDE. Could you check it?

paulinaurbas commented:
1. Duplicate bug report, fixed in https://github.com/paulinaurbas/puppy.io/pull/21
2. Good catch but it is not a bug 😄 This is correct library behaviour cased by fact that I added 3 the same pictures with same url in mock: 
listOfUrls: [(state).dogOffer.photoUrl, (state).dogOffer.photoUrl, (state).dogOffer.photoUrl],

Any way I corrected dog offer model and I removed duplicates of photoUrls, also in https://github.com/paulinaurbas/puppy.io/pull/21 😉
